src/main/scala/lenthall/spray/SprayCanHttpService.scala

```diff
@@ -0,0 +1,167 @@
+package lenthall.spray
+
+import java.net.{InetAddress, InetSocketAddress}
+
+import akka.actor._
+import akka.io.Tcp.Unbound
+import akka.io.{IO, Inet}
+import akka.util.Timeout
+import spray.can.Http
+import spray.can.Http.Bound
+import spray.can.server.ServerSettings
+import spray.io.ServerSSLEngineProvider
+
+import scala.collection.immutable
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ExecutionContext, Future, Promise, TimeoutException}
+
+/**
+ * Adds bind and unbind to http services running on spray-can.
+ *
+ * {{{
+ * val httpServiceActorRef: ActorRef = ...
+ * httpServiceActorRef.bind(LoopbackAddress, 8080) onSuccess {
+ *   case boundListener =>
+ *     boundListener.unbind() // Unbind the listener
+ * }
+ * }}}
+ */
```

Comment:
Since Lenthall ought to be a public library, it'd be nice if the README had examples of how to use each component of Lenthall...  I'm not saying it's a blocker for this PR, but it would help me understand this stuff better.